docs/components/progress.md

```diff
@@ -60,7 +61,15 @@ Uses a gradient to create a striped effect.
 
 The striped gradient can also be animated. Add `.progress-animated` to `.progress` to animate the stripes right to left via CSS3 animations.
 
-**Animated progress bars do not work in IE9 and Opera 12** – as they don't support CSS3 animations – **nor in IE10+ and Microsoft Edge** – as they currently don't support CSS3 animations on the [`::-ms-fill` pseudo-element](https://msdn.microsoft.com/en-us/library/windows/apps/hh465757.aspx).
+**Most browser don't support CSS3 animations on the `::-*-fill` pseudo-element so apply `.progress` on `<div>`s to animate the stripes 
```

Comment:
I'd rather have browsers that don't support it listed, instead of the vagueness of "most browsers". Also, the text here should clarify that you're effectively NOT using `<progress>` then, and instead faking it with `<div>` and `<span>`